cmd/istio_ca/main.go

```diff
@@ -46,6 +47,11 @@ var (
 	selfSignedCA = flag.Bool("self-signed-ca", false,
 		"Indicates whether to use auto-generated self-signed CA certificate. "+
 			"When set to true, the '-ca-cert' and '-ca-key' options are ignored.")
+
+	caCertTTL = flag.Duration("ca-cert-ttl", 240*time.Hour,
```

Comment:
Just curious, what will happen when caCert expires while proxy cert is still valid? Would mTLS break?